src/routes/Forms/components/Forms.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+import {getFormData} from '../modules/account'
+
+class Form extends Component {
```

Comment:
Since this is a stateless component, I suppose you can just write an arrow function for this? Something like
`const Form = () => {}`
(details not included)
Your method `printFormData` isn't actually interacting with the component so I think arrow function is actually encouraged. Correct me if I'm wrong